lightwood/data/timeseries_transform.py

```diff
@@ -132,11 +133,14 @@ def transform_timeseries(
         combined_df = pd.DataFrame(combined_df[combined_df['__mdb_make_predictions'].astype(bool).isin([True])])
         del combined_df['__mdb_make_predictions']
 
-    if len(combined_df) == 0:
-        raise Exception(f'Not enough historical context to make a timeseries prediction. Please provide a number of rows greater or equal to the window size. If you can\'t get enough rows, consider lowering your window size. If you want to force timeseries predictions lacking historical context please set the `allow_incomplete_history` advanced argument to `True`, but this might lead to subpar predictions.') # noqa
+    if len(combined_df) < n_groups * tss.window:
```

Comment:
will this actually work though? As in, shouldn't we test to see that forecasting with incomplete history doesn't result in a crash at the encoder level. I'm still not 100% up to date on the RNN encoder design so maybe if it's "obvious" it would work, forget about the test, since it's kind of an edge case anyway.

Also `log.warning` rather than `log.info`... maybe (?)

Reply:
The encoders should be able to handle it, because by this point the data has been reshaped into a regular size (even if the historical context has lots of `None`s). I will add a test though, to be sure.